types/node-telegram-bot-api/index.d.ts

```diff
@@ -1386,6 +1403,61 @@ declare class TelegramBot extends EventEmitter {
         fileOptions?: TelegramBot.FileOptions,
     ): Promise<TelegramBot.Message>;
 
+    getStickerSet(
+        name: string,
+        options?: {},
+    ): Promise<TelegramBot.StickerSet>;
+
+    getCustomEmojiStickers(
+        customEmojiIds: string[],
+        options?: {},
+    ): Promise<TelegramBot.Sticker[]>;
+
+    uploadStickerFile(
+        userId: string,
+        pngSticker: string | Stream | Buffer,
+        options?: {},
+        fileOptions?: TelegramBot.FileOptions,
+    ): Promise<TelegramBot.File>;
+
+    createNewStickerSet(
+        userId: string,
+        name: string,
+        title: string,
+        emojis: string,
+        pngSticker?: string | Stream | Buffer,
```

Comment:
Although telegram do support using one of png_sticker, tgs_sticker or webm_sticker, the [implementation](https://github.com/yagop/node-telegram-bot-api/blob/ccdd14670b10548eda32397cff3f3ed678991f0a/src/telegram.js#L2231) only support png_sticker. Could you consider making the parameter `pngSticker` required?